main.cpp

```diff
@@ -331,8 +332,33 @@ static void cipher_examples(void)
     }
 }
 
+static void fake_set_initial_nvseed(void)
+{
+    /* mbedtls_psa_inject_entropy() depends on both MBEDTLS_ENTROPY_NV_SEED and
+     * MBEDTLS_PSA_HAS_ITS_IO being enabled by the Mbed TLS configuration
+     * system. */
+#if defined(MBEDTLS_ENTROPY_NV_SEED) && defined(MBEDTLS_PSA_HAS_ITS_IO)
```

Comment:
This may be wrong in case of 2 cores.
for example the Psoc6:
the M4 core will not have NV_SEED (so this function will not compile) but defined but the M0 will have it.

Reply:
Agreed. However, we may want to comment this is a generic implementation for single core systems and may need to be implemented on cases-by case basis?